spacetemplate/assets/base.yaml

```diff
@@ -47,6 +47,26 @@ work_item_types:
         - in progress
         - resolved
         - closed
+    "system.metastate":
+      label: Meta State
+      description: The meta-state of the work item
+      read_only: yes
+      required: no
+      type:
+        simple_type:
+          kind: enum
+        base_type:
+          kind: string
+        # This will allow other WITs to overwrite the values of the state.
+        rewritable_values: yes
```

Comment:
@michaelkleinhenz I thought the whole point of the meta-state was to provide a generic mapper of state over the the whole system. Do you still want to let other WITs redefine those values?

Reply:
mmh..no. That should not be allowed.